src/components/Header_type_1.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { routeTo } from 'redux-router-kit';
+import { getTranslator } from '../reducers/translation';
+import { RoundHeaderButton } from './Buttons';
+
+import styles from './Header_type_1.scss';
+import common from '../styles/common.scss';
+
+const HeaderInitialComponent = ({ i18n, targetTo }) =>
+  (<header>
+    <div className={common.widthLimiter}>
+      <div className={styles.header}>
+        <Grid fluid>
+          <Row>
+            <Col lg={1} className={styles.logoContainer}>
+              <div className={styles.logo} />
+            </Col>
+            <Col lg={8} className={styles.mainArea}>
+              <div className={styles.title}>
+                {i18n('Neufund Platform')}
+              </div>
+              <div className={styles.status}>
+                {i18n('Log in')}
+              </div>
+            </Col>
+            <Col lg={3} className={styles.createButton}>
+              <RoundHeaderButton
+                onClick={(event) => {
+                  event.preventDefault();
+                  targetTo('/register');
+                }}
+                style={{ display: 'inline-block' }}
```

Comment:
Adding @Moejoe90  as this might be interesting also to you. 
Our current buttons have `display: block` by default. I wasn't sure if it was something you planned when implementing those elements.
If we want to copy existing patterns then for example material-ui buttons are indeed `inline-block`. I can create PR for this. 

Setting this through styles at our current implementation is not possible. If I would do this
`<RoundHeaderButton className={styles.ownStyle}>...</RoundHeaderButton>`
Rendered element would receive styles only from my `ownStyle` class.  

Anyway if we change implementation that button would receive styles from both style-sheets we would have `display` property defined on same level in two different files. Outcome which one will be used by browser would be not so obvious - css path length and rule order would define that. Or `!important` keyword. 

I don't see straightforward elegant solution here.